decentraland/unity-explorer
# fix: UI cleanup for quest related elements

## What does this PR change?

This chunky PR cleans up different UI elements related to the quest shape up, from the sidebar to some map components, the majority of issues and improvements are listed here https://docs.google.com/spreadsheets/d/1eTpbIfSQVHCVeSmkFB9lufnm-pbsHfyJXZANwzlIJxM/edit?pli=1&gid=1297425249#gid=1297425249

...

## How to test the changes?



1. Launch the explorer
2. Play around with the sidebar
3. Verify it works fine

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
